docs/sources/httppoller.md

```diff
@@ -0,0 +1,47 @@
+# Event Source for HTTP Polling
+
+This event source launches periodic HTTP requests against an external system endpoint, turning received requests into [CloudEvents][ce] to be consumed by other TriggerMesh components.
+
+## Prerequisites
+
+- An external system that exposes an HTTP endpoint.
+- When using HTTP basic authentication, a secret containing the password.
+
+## Deploying an Instance of the HTTP Poller Source
+
+Open the Bridge creation screen and add a source of type `HTTP Poller`.
+
+In the Source creation form add the following information:
+
+- **Name**: all Triggermesh components need a unique name per namespace.
+- **Broker**: request converted into [CloudEvents][ce] will be sent to this location.
+- **EventType**: string that identifies the purpose for all CloudEvent messages produced from this source.
+- **EventSource**: (optional) string that identifies the origin for all CloudEvent messages produced from this source.
+- **Enpoint**: URL location for the remote service to be polled.
+- **Method**: HTTP method.
+- **Frequency**: periodicity for requests formatted as [Go duration][go-duration].
```

Comment:
IMHO it should have been named "Interval", but ok...

Reply:
that would be a change for the CRD.
Since controller and frontend are merged that will need to go through all other layers.

I would suggest to merge this, then I'll try to have those 3 changes (sources/fe/docs) in place